src/Resources/public/js/startup.js

```diff
@@ -209,7 +209,7 @@ pimcore.plugin.customermanagementframework = Class.create({
         var object = event.detail.object;
         var type = event.detail.type;
 
-        if ("object" === type && object.data.general.o_className === pimcore.settings.cmf.customerClassName && pimcore.globalmanager.get("user").isAllowed(ActivityView.config.PERMISSION)) {
+        if ("object" === type && object.data.general.className === pimcore.settings.cmf.customerClassName && pimcore.globalmanager.get("user").isAllowed(ActivityView.config.PERMISSION)) {
```

Comment:
@dvesh3 what should we do in these cases where the `o_` prefix is being used in js?
This could be a potential problem if we are just going to replace the o_ prefixes without some kind of bc layer to support both pimcore 10.5 and 11.